src/backend/dx12/src/device.rs

```diff
@@ -1904,6 +1952,10 @@ impl d::Device<B> for Device {
             } else {
                 None
             },
+            dxgi_format: image.dxgi_format,
+            num_levels,
```

Comment:
any reason these can't be stored as `Range` as opposed to the tuples?

Reply:
wanted to preserve the `Copy` derive

Reply:
FYI, I'm not really a fan of getting `Copy` on everything that's copyable. Rust programs are already notorious for spending most of their run times copying bits around (as opposed to be doing actual work, lol), and making that cost implicit doesn't help. At least when you are forced to do a `clone()` that makes you consider using a reference here and there and minimize the copy costs.